### PR TITLE
fix(blog): Minor grammatical changes

### PR DESCRIPTION
**Pull request by chenxsan**

### CODE REVIEW

File: src/content/blog/2020-10-10-webpack-5-release.md [hunk 1]
```diff
@@ -1393,7 +1394,7 @@ These dependencies are cheaper to process and webpack uses them when possible
 - DependencyReference now takes a function to a module instead of a Module
 - HarmonyImportSpecifierDependency.redirectedId removed
     - __MIGRATION__: Use `setId` instead
-- acorn 5 -> 7
+- acorn 5 -> 8
```
Comment (chenxsan) [inline src/content/blog/2020-10-10-webpack-5-release.md:1397]: Check https://github.com/webpack/webpack/blob/master/package.json#L14

**Comment by chenxsan** [comment]
CLA is buggy.

**Review by evilebottnawi — APPROVED** [review]
Ignore CLA

**Review by montogeek — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Review by jeffin143 — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]

**Review by jamesgeorge007 — APPROVED** [review]

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**